data/groups.json

```diff
@@ -78,22 +78,19 @@
       "cad789",
       "cryptostorms",
       "dafricash",
-      "devasive",
```

Comment:
Source?

Reply:
Can't he/she come here and write it as Samuray2016 did? Or write it in the blockchain from his address?

Reply:
asked

Reply:
I can confirm that I am no longer a member of the AFT group